examples/text/plot_document_classification_20newsgroups.py

```diff
@@ -186,52 +270,48 @@ def benchmark(clf):
     print(metrics.confusion_matrix(y_test, pred))
```

Comment:
Let's remove the text classification reports and the text classification matrix in the benchmark tool. We already demonstrated how to display the confusion matrix for a given model in the first part of the notebook.

Reply:
We do not use the classification report on the demo with `RidgeClassifier`, should I add it there?